modules/install/pages/upgrade.adoc

```diff
@@ -150,11 +150,11 @@ Note, however, that _Graceful Failover_ for Data Service nodes, with _Delta Reco
 [#node-certificate-validation]
 == Node Certificate Validation
 
-In Couchbase Enterprise Server Version 7.2+, the node-name _must_ be correctly identified in the node certificate as a Subject Alternative Name.
+In Couchbase Enterprise Server Version 7.2 or later, the node-name _must_ be correctly identified in the node certificate as a Subject Alternative Name.
 If such identification is not correctly configured, failure may occur when uploading the certificate, or when attempting to add or join the node to a cluster.
 
-When upgrade is performed from a pre-7.2 cluster to a 7.2+ cluster, if a new, 7.2 node is introduced to the cluster without the node-name being correctly configured in its certificate, a warning is provided.
-Likewise, if a new certificate is uploaded onto a 7.2 node, a warnng is provided.
+When upgrade is performed from a pre-7.2 cluster to a 7.2+ cluster, if a new, 7.2+ node is introduced to the cluster without the node-name being correctly configured in its certificate, a warning is provided.
```

Comment:
I have resorted to a trial addition of the table, to answer these questions. I'm finding it difficult to explain the process effectively at a high level -- since there are so many different scenarios, and generalisations can't easily be made. So perhaps being fully specific is the way forward (?) I'm not sure, because it does seem a lot of info. Let me know what you think. Or, if there is some other way of subsetting the information effectively, let me know and I'll try it.

Reply:
Tony,

In this case, a failure will happen not a warning.